yarn-project/scripts/src/benchmarks/aggregate.ts

```diff
@@ -167,10 +167,19 @@ function processTxSequencerProcessingStats(entry: TxSequencerProcessingStats, re
 /** Process a tree insertion event and updates results */
 function processTreeInsertion(entry: TreeInsertionStats, results: BenchmarkCollectedResults) {
   const bucket = entry.batchSize;
+  const depth = entry.treeDepth;
   if (entry.treeType === 'append-only') {
-    append(results, 'batch_insert_into_append_only_tree_ms', bucket, entry.duration);
+    if (depth === 16) {
+      append(results, 'batch_insert_into_append_only_tree_16_depth_ms', bucket, entry.duration);
+    } else if (depth === 32) {
+      append(results, 'batch_insert_into_append_only_tree_32_depth_ms', bucket, entry.duration);
+    }
```

Comment:
:exclamation: This ignores every other tree depth

We should revisit this once the metrics definition supports dynamic metric names